content/applications/sales/point_of_sale/overview/getting_started.rst

```diff
@@ -80,25 +93,50 @@ Return and refund products
 
 To return a product and make a refund,
 
-#. from the **POS dashboard**, **open a session** and select the returned product;
-#. click :guilabel:`Qty` and :guilabel:`+/-` to enter the quantity of returned products;
+#. :ref:`start a session <session>` from the **POS dashboard**;
+#. click :guilabel:`Refund` and select the corresponding order;
+#. select the product and the quantity to refund using the keypad;
+#. click :guilabel:`Refund` to go back to the previous screen;
 #. once the order is completed, click :guilabel:`Payment` to proceed to the refund;
 #. click :guilabel:`Validate` and :guilabel:`New Order` to move on to the next customer.
 
+.. note::
+   - You can filter the **orders list** by :guilabel:`Receipt Number`, :guilabel:`Date` or
+     :guilabel:`Customer` using the search bar.
+   - You can also refund a product by selecting the returned product from an open session, and
+     setting a negative quantity that equals the number of returned products. To do so, click
+     :guilabel:`Qty` and :guilabel:`+/-`, followed by the quantity of returned products.
+
 Close the POS session
 ---------------------
 
-To close your session,
+To close your session, click :guilabel:`Close` in the upper right corner of your screen; doing so
+opens the :guilabel:`Closing Control` pop-up screen. From this screen, you can retrieve various
+information:
 
-#. go back to the POS dashboard by clicking :guilabel:`Close` and :guilabel:`Confirm` in the upper
-   right corner of your screen;
-#. then, click :guilabel:`Close` and proceed to the **closing control**;
-#. click the :guilabel:`Payments` smart button to know what amount of cash should be in your cash
-   drawer;
-#. once the control is done, click :guilabel:`Close session & post entries`;
-#. the status goes automatically from :guilabel:`In Progress` to :guilabel:`Closed & Posted`.
+- the number of orders made and the total amount made during the session;
+- the total of all the transactions made during the session grouped by payment method;
+- the expected amounts grouped by payment method.
+
+Before closing this window, count your cash using the calculator icon. Doing so opens a pop-up
+window that computes the total amount in the cash drawer depending on the coins and bills counted
+and added manually. Then, click :guilabel:`Confirm` or :guilabel:`Discard` to close the window. The
+computed amount is set in the :guilabel:`Counted` column, and the :guilabel:`Money Details` are
+specified in the **Notes** section.
+
```

Comment:
I would move the image here as it illustrates the above paragraph 
.. image:: getting_started/closing-control.png
   :align: center
   :alt: How to close a POS session.